app/models/grouping.rb

```diff
@@ -10,4 +10,12 @@ class Grouping < ApplicationRecord
   def children
     desks + child_groupings
   end
+
+  def type
+    'grouping'
+  end
+
+  def occupied
```

Comment:
Since you return a boolean, let's follow the `?` ruby method name pattern: `def occupied?`